packages/opa-react/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*.ts", "tests/**/*.ts"],
+  "include": ["src/**/*.ts", "tests/**/*.ts", "tests/setup.ts"],
```

Comment:
tried explicitly including it, in case it wasn't -- like assumed -- matched by the glob preceding it